lib/PicoDb/Table.php

```diff
@@ -675,8 +674,7 @@ public function innerJoinSubquery(Table $subQuery, string $alias, string $foreig
 
         $this->joinValues = array_merge(
             $this->joinValues,
-            $subQuery->getConditionBuilder()->getValues(),
-            $subQuery->getAggregatedConditionBuilder()->getValues()
+            $subQuery->getValues()
```

Comment:
Whats the difference here for the condition builders values vs the base values?

Reply:
@b-hayes `Table::getValues()` is a combination of the two lines I've replaced _plus_ the values bound for any JOINs (which was missing and causing our issues). So in order, it collects values for:

1. JOINs
2. WHERE conditions
3. HAVING conditions

Reply:
Ah yeah just looked at the code for it its basically doing the same thing but inclused join values.